go/store/nbs/mem_table.go

```diff
@@ -153,8 +153,19 @@ func (mt *memTable) getMany(ctx context.Context, reqs []getRecord, foundChunks c
 	return remaining
 }
 
-func (mt *memTable) getManyCompressed(ctx context.Context, reqs []getRecord, foundCmpChunks chan<- chunks.Chunkable, wg *sync.WaitGroup, ae *atomicerr.AtomicError, stats *Stats) bool {
-	panic("not implemented")
+func (mt *memTable) getManyCompressed(ctx context.Context, reqs []getRecord, foundCmpChunks chan<- CompressedChunk, wg *sync.WaitGroup, ae *atomicerr.AtomicError, stats *Stats) bool {
```

Comment:
@bheni We implemented this with `ChunkToCompressedChunk`.

Reply:
should be fine.  Not optimal, but we aren't using memtable for push and pull anymore, so memTable.getManyCompressed may not even be used in the current codebase.